frontend/src/components/Header/Header.js

```diff
@@ -7,8 +7,11 @@ import HeaderDropdown from '../HeaderDropdown/HeaderDropdown';
 import './Header.css';
 
 const Header = ({
-  history, user, setJwt, showSearchBar,
+  history, user, setJwt, showSearchBar, scrollDown,
 }) => {
+  const handleClick = () => {
```

Comment:
I think making this name more descriptive would be good, too. Lots of things you can click on the header, and this only handles one of them.